packages/block-editor/src/components/global-styles/color-panel.js

```diff
@@ -230,6 +230,11 @@ function ColorPanelDropdown( {
 							{ 'is-open': isOpen }
 						),
 						'aria-expanded': isOpen,
+						'aria-label': sprintf(
```

Comment:
Extracting out this change plus necessary updates to E2E tests over in https://github.com/WordPress/gutenberg/pull/50307 to reduce a bit of clutter